code/controllers/configuration.dm

```diff
@@ -254,7 +254,7 @@
 	var/error_silence_time = 6000 // How long a unique error will be silenced for
 	var/error_msg_delay = 50 // How long to wait between messaging admins about occurrences of a unique error
 
-	var/arrivals_shuttle_dock_window = 55	//Time from when a player late joins on the arrivals shuttle to when the shuttle docks on the station
+	var/arrivals_shuttle_dock_window = 100	//Time from when a player late joins on the arrivals shuttle to when the shuttle docks on the station

```

Comment:
And if you just want to change the default for other servers you have to change in the actual config.txt